monitoring/terraform/stack/post_to_slack.tf

```diff
@@ -1,69 +1,69 @@
+    BITLY_ACCESS_TOKEN        = var.bitly_access_token
   }
 
-  alarm_topic_arn = "${var.lambda_error_alarm_arn}"
+  alarm_topic_arn = var.lambda_error_alarm_arn
 
   log_retention_in_days = 30
 }
 
 module "trigger_post_to_slack_dlqs_not_empty" {
-  source = "git::https://github.com/wellcometrust/terraform.git//lambda/trigger_sns?ref=v1.0.0"
+  source = "../modules/lambda_trigger_sns"
 
-  lambda_function_name = "${module.lambda_post_to_slack.function_name}"
-  lambda_function_arn  = "${module.lambda_post_to_slack.arn}"
-  sns_trigger_arn      = "${var.dlq_alarm_arn}"
+  lambda_function_name = module.lambda_post_to_slack.function_name
+  lambda_function_arn  = module.lambda_post_to_slack.arn
+  sns_trigger_arn      = var.dlq_alarm_arn
 }
 
 module "trigger_post_to_slack_server_error_gateway" {
-  source = "git::https://github.com/wellcometrust/terraform.git//lambda/trigger_sns?ref=v1.0.0"
+  source = "../modules/lambda_trigger_sns"
 
-  lambda_function_name = "${module.lambda_post_to_slack.function_name}"
-  lambda_function_arn  = "${module.lambda_post_to_slack.arn}"
-  sns_trigger_arn      = "${var.gateway_server_error_alarm_arn}"
+  lambda_function_name = module.lambda_post_to_slack.function_name
+  lambda_function_arn  = module.lambda_post_to_slack.arn
+  sns_trigger_arn      = var.gateway_server_error_alarm_arn
 }
 
 module "trigger_post_to_slack_lambda_error" {
-  source = "git::https://github.com/wellcometrust/terraform.git//lambda/trigger_sns?ref=v1.0.0"
+  source = "../modules/lambda_trigger_sns"
 
-  lambda_function_name = "${module.lambda_post_to_slack.function_name}"
-  lambda_function_arn  = "${module.lambda_post_to_slack.arn}"
-  sns_trigger_arn      = "${var.lambda_error_alarm_arn}"
+  lambda_function_name = module.lambda_post_to_slack.function_name
+  lambda_function_arn  = module.lambda_post_to_slack.arn
+  sns_trigger_arn      = var.lambda_error_alarm_arn
 }
 
 resource "random_id" "statement_id" {
   keepers = {
-    aws_sns_topic_subscription = "${aws_sns_topic_subscription.subscribe_lambda_to_cloudfront_errors.id}"
+    aws_sns_topic_subscription = aws_sns_topic_subscription.subscribe_lambda_to_cloudfront_errors.id
   }
 
   byte_length = 8
 }
 
 resource "aws_lambda_permission" "allow_sns_cloudfront_trigger" {
-  statement_id  = "${random_id.statement_id.hex}"
+  statement_id  = random_id.statement_id.hex
   action        = "lambda:InvokeFunction"
-  function_name = "${module.lambda_post_to_slack.arn}"
+  function_name = module.lambda_post_to_slack.arn
   principal     = "sns.amazonaws.com"
-  source_arn    = "${var.cloudfront_errors_topic_arn}"
-  depends_on    = ["aws_sns_topic_subscription.subscribe_lambda_to_cloudfront_errors"]
+  source_arn    = var.cloudfront_errors_topic_arn
+  depends_on    = [aws_sns_topic_subscription.subscribe_lambda_to_cloudfront_errors]
 }
 
 resource "aws_sns_topic_subscription" "subscribe_lambda_to_cloudfront_errors" {
-  provider = "aws.us_east_1"
+  provider = aws.us_east_1
```

Comment:
What's the reason for these living in Virginia?

Reply:
CloudFront lives in Virginia.

Reply:
CloudFront.